management/prompt.go

```diff
@@ -19,10 +19,6 @@ type PromptManager struct {
 	*Management
 }
 
-type promptCustomText struct {
```

Comment:
This was never used anywhere and it's also a private type, so it can safely be removed.